src/components/Facility/AssetCreate.tsx

```diff
@@ -513,400 +406,540 @@ const AssetCreate = (props: AssetProps) => {
-                      <FieldErrorText
-                        error={state.errors.last_serviced_on}
-                      ></FieldErrorText>
-                    </div>
+                      {sectionTitle("Service Details")}
 
-                    {/* Notes */}
-                    <div
-                      className="col-span-6 mt-6"
-                      ref={fieldRef["notes"]}
-                      data-testid="asset-notes-input"
-                    >
-                      <TextAreaFormField
-                        name="notes"
-                        label={t("notes")}
-                        placeholder={t(
-                          "Eg. Details on functionality, service, etc.",
-                        )}
-                        value={notes}
-                        onChange={(e) => setNotes(e.value)}
-                        error={state.errors.notes}
-                      />
+                      {/* Serviced On */}
+                      <div
+                        className="col-span-6 sm:col-span-3"
+                        data-testid="asset-last-serviced-on-input"
+                      >
+                        <FormField
+                          control={form.control}
+                          name="serviced_on"
+                          render={({ field }) => (
+                            <FormItem className="flex flex-col">
+                              <FormLabel className="mt-3">
+                                Last Serviced On
+                              </FormLabel>
+                              <Popover>
+                                <PopoverTrigger asChild>
+                                  <FormControl>
+                                    <Button
+                                      variant={"outline"}
+                                      className={cn(
+                                        "w-[240px] pl-3 text-left font-normal",
+                                        !field.value && "text-muted-foreground",
+                                      )}
+                                    >
+                                      {field.value ? (
+                                        format(field.value, "PPP")
+                                      ) : (
+                                        <span>Pick a date</span>
+                                      )}
+                                      <CareIcon
+                                        icon="l-calender"
+                                        className="ml-auto h-4 w-4 opacity-50"
+                                      />
+                                    </Button>
+                                  </FormControl>
+                                </PopoverTrigger>
+                                <PopoverContent
+                                  className="w-auto p-0"
+                                  align="start"
+                                >
+                                  <Calendar
```

Comment:
There's issues with both rendering and behavior for the calendar. If you aren't able to solve it, stick with existing DateFormField for now; we will need to migrate or create a similar component with Shadcn's styles eventually.

Reply:
will be working on it..

Reply:
> There's issues with both rendering and behavior for the calendar. If you aren't able to solve it, stick with existing DateFormField for now; we will need to migrate or create a similar component with Shadcn's styles eventually.

@jacobjeevan tried to align it properly, but i think its a problem in the template itself,  hope they fix it soon🤞
I've switched to old `DateFormField` for Expiry and Last Service Date fields wrapped in ShadCN FormControl, please review the changes.